src/screens/DiscoveryScreenV2/elements/DiscoveryItemList.js

```diff
@@ -154,9 +168,21 @@ const DomainList = (props) => {
 
 // data needed name, description, image
 const styles = StyleSheet.create({
+  communityTextContainer: {
+    flexDirection: 'row',
+    alignItems: 'center'
+  },
+  communityText: {
+    color: COLORS.blueOnboarding,
+    fontFamily: fonts.inter[400],
+    fontSize: 12,
+    flexWrap: 'wrap',
+    lineHeight: 18,
+    marginTop: 4
+  },
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

New styles have been added for the community text container. Ensure that these styles are applied correctly across different screen sizes and resolutions.



<!-- This is an auto-generated comment by OSS CodeRabbit -->